### PR TITLE
feature: add task update actions in client pkg of golang sdk

### PR DESCRIPTION
**Pull request by zgfh**
### Ⅰ. Describe what this PR did
feature: add task update actions in client pkg of golang sdk

### Ⅱ. Does this pull request fix one issue?

this fixes one part of #366

### Ⅳ. Describe how to verify it
none

### Ⅴ. Special notes for reviews
none

### CODE REVIEW

File: client/task_update.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import (
+	"context"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+)
+
+// TaskUpdate update a task in supernode.
+func (client *APIClient) TaskUpdate(ctx context.Context, id string, config *types.TaskUpdateRequest) error {
+	resp, err := client.put(ctx, "/tasks"+id, nil, config, nil)
```
Comment (allencloud) [inline client/task_update.go]: I think the url path should be `"/tasks/"+id`. You have missed one `/`.

File: client/task_update.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+package client
+
+import (
+	"context"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+)
+
+// TaskUpdate update a task in supernode.
```
Comment (allencloud) [inline client/task_update.go]: TaskUpdate updates a task in supernode.
please use updates.

**Comment by allencloud** [comment]
Could you help to add the mock test of unit test of Client Functions? Like the file: https://github.com/dragonflyoss/Dragonfly/blob/master/client/preheat_info_test.go @zgfh

File: client/task_update.go (hunk 2 above)
Comment (zgfh) [reply client/task_update.go]: updated

File: client/task_update.go (hunk 1 above)
Comment (zgfh) [reply client/task_update.go]: sorry,updated

**Comment by allencloud** [comment]
LGTM

**Pull request merged**